src/core/RecyclerListView.tsx

```diff
@@ -212,6 +213,13 @@ export default class RecyclerListView extends React.Component<RecyclerListViewPr
                 const offset = this.props.contextProvider.get(uniqueKey);
                 if (typeof offset === "number" && offset > 0) {
                     this._initialOffset = offset;
+                    if (this.props.onRecreate) {
```

Comment:
Why pass null, just pass a map since this in not a scrollevent, also merge stableIds branch before this